content/spin/quickstart.md

```diff
@@ -77,7 +77,9 @@ Unzip the binary release and place the `spin.exe` in your system path.
 
 ### Install a Template
 
-The quickest and most convenient way to start a new application is to use a Spin template.  Let's install the templates for your preferred language.
+The quickest and most convenient way to start a new application is to use a Spin template.
+
+Let's install the templates for your preferred language.
```

Comment:
nit: suggest combining this with the previous paragraph, as it was before (I don't feel strongly about it, just the change feels a bit superfluous now that there's no longer the extra words about whether or not you need to do it)